cmd/release-controller/http.go

```diff
@@ -725,8 +725,11 @@ func (c *Controller) httpReleaseInfo(w http.ResponseWriter, req *http.Request) {
 	}
 	if len(upgradesTo) > 0 {
 		sort.Sort(newNewestSemVerFromSummaries(upgradesTo))
-		fmt.Fprintf(w, `<p id="upgrades-from">Upgrades from:</p>`)
-		fmt.Fprintf(w, "<div class=\"alert alert-warning\">Untested upgrades: %s</div><ul>", strings.Join(missingUpgrades, ", "))
+		if missingUpgrades != nil {
```

Comment:
If missing upgrades is an array, you should check by looking for `len(missingUpgrades) > 0`.  An empty array and a nil array are the same.

Reply:
Good to know!  Thanks for the pointer!